vci-puppeteer/vci-approve-variants.js

```diff
@@ -28,6 +29,33 @@ const DOMAIN_CONFIG = {
 const CREDENTIALS_DATA = fs.readFileSync('credentials.json');
 const CREDENTIALS = JSON.parse(CREDENTIALS_DATA);
 
+class ButtonMissingError extends Error {
+  constructor(message) {
+    super(message); // (1)
+    this.name = "ButtonMissingError"; // (2)
+  }
+}
+
+// Keept track of stats on the run to print out at the end or on exit.
+let runStatistics = {
```

Comment:
Sam, we made this global now (outside of a function, before it was in the function making the var local to that function). Now we have access to it wherever, namely at the block where we intercept an exit ctl-C command  (defined in `process.on( 'SIGINT',`).